src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,55 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int SIZE = 10;
+    private Object[] keys;
+    private Object[] values;
+    private int size;
+
+    public StorageImpl() {
+        this.keys = new Object[SIZE];
+        this.values = new Object[SIZE];
```

Comment:
```suggestion
        this.keys = (K[]) new Object[SIZE];
        this.values = (V[]) new Object[SIZE];
```
After this you will be able to remove casting from methods